tests/_data/container.php

```diff
@@ -85,6 +85,15 @@ function (ServerRequestInterface $request, ResponseInterface $response) {
             }
         );
 
+        $app->post(
```

Comment:
I think like we could use existing `POST /rest` for this.  
No need to create new route.